resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesConf.scala

```diff
@@ -166,6 +168,18 @@ private[spark] class KubernetesExecutorConf(
     KubernetesVolumeUtils.parseVolumesWithPrefix(sparkConf, KUBERNETES_EXECUTOR_VOLUMES_PREFIX)
   }
 
+  def checkExecutorEnv(executorEnvKey: String): Boolean = {
```

Comment:
private?

Reply:
Thanks for your reminder, It has been resolved.